src/Bicep.Core/Syntax/MetadataDeclarationSyntax.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+//using System.Collections.Immutable;
+using System.Linq;
+using Bicep.Core.Navigation;
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.Syntax
+{
+    public class MetadataDeclarationSyntax : StatementSyntax, ITopLevelNamedDeclarationSyntax
+    {
+        public MetadataDeclarationSyntax(IEnumerable<SyntaxBase> leadingNodes, Token keyword, IdentifierSyntax name, SyntaxBase assignment, SyntaxBase value)
```

Comment:
Please also update the following files

Grammar spec:
* ./docs/grammar.md

Highlight libraries (should just need to add to the list of keywords hardcoded in the following files):
* src/highlightjs/src/bicep.ts
* src/monarch/src/bicep.ts
* src/textmate/src/bicep.ts